File: 7KYU/Sum of Odd Numbers/solution.cs
using System;
public static class Kata
{
	public static long rowSumOddNumbers(long n)
	{	
		var last = (n * n) + (n - 1);
		var start = last - (2 * (n -1));
		return ((start + last) / 2) * n; 
	}
}

File: 7KYU/Sum of Odd Numbers/solution.java
class RowSumOddNumbers {
    public static int rowSumOddNumbers(int n) {
        var last = (n * n) + (n - 1);
        var start = last - (2 * (n - 1));
        return ((start + last) / 2) * n;
    }
}